### DISCUSSION TITLE
doesn't work code assist and analysis

### DISCUSSION THREAD

**Discussion by seiswrd** [BowlerHatLLC/vscode-as3mxml#760] [Q&A] [answered]
hi!
what is this error message? doesn't work code assist.
someone please help me

windows11
visual studio code 1.92.2
flex sdk version : adobe flex 3.2

[{
	"resource": "/PROJECT_DIR/",
	"owner": "_generated_diagnostic_collection_name_#2",
	"code": "1319",
	"severity": 8,
	"message": "Invalid version specified, '10'.\n",
	"startLineNumber": 1,
	"startColumn": 1,
	"endLineNumber": 1,
	"endColumn": 1
}]

**Comment by seiswrd** [comment]
it work on apache flex 4.16.1 version

**Comment by joshtynjala** [comment, marked as answer]
[FAQ: Invalid version specified error](https://github.com/BowlerHatLLC/vscode-as3mxml/wiki/Common-issues#invalid-version-specified-90124-compiler-error)

**Reply by seiswrd** [reply]
thank you!
